### PR TITLE
fix: Transparent View Controller Navigation Bars - IC - 58

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we fix a couple of navigation bars being transparent in certain scenarios where use would tap on `started using a new device` as shown in the screenshot below. 

testy

Screenshot 2023-04-03 at 11 19 08

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by KaterinaWire — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift [hunk 1]
```diff
@@ -347,7 +347,7 @@ final class ZClientViewController: UIViewController {
     /// - Parameter conversation: conversation to open
     func openDetailScreen(for conversation: ZMConversation) {
         let controller = GroupDetailsViewController(conversation: conversation)
-        let navController = controller.wrapInNavigationController()
+        let navController = controller.wrapInNavigationController(setBackgroundColor: true)
```
Comment (johnxnguyen) [inline wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift:350]: Is there any case that we would set background color false?

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift:350]: @johnxnguyen Good question. I was thinking about that while I was making those changes. I couldn't think of any. 

How do you think we should proceed?

Comment (johnxnguyen) [reply wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift:350]: Maybe it solves this transparent issue one and for all, but maybe it introduces some other bugs. If I remember correctly, it had something to do with iOS versions. It doesn't hurt to try and run the app on different iOS versions to see if everything is fine.

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift:350]: I can definitely do that. Can we merge this PR? And I will create a ticket for further investigating this.

**Review by MarcinR — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift (hunk 1 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift:350]: @johnxnguyen I will create a ticket and further investigate!

**Pull request merged**